### PR TITLE
wget2 1.99.1 (new formula)

### PR DESCRIPTION
**Pull request by jakob-stoeck**
GNU Wget2 is the successor of GNU Wget, a file and recursive website downloader.

Designed and written from scratch it wraps around libwget, that provides the basic functions needed by a web client.

Wget2 works multi-threaded and uses many features to allow fast operation.

In many cases Wget2 downloads much faster than Wget1.x due to HTTP zlib compression, parallel connections and use of If-Modified-Since HTTP header.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wget2.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Wget2 < Formula
+  desc "Multithreaded metalink/file/website downloader"
+  homepage "https://gitlab.com/gnuwget/wget2/"
+  url "https://alpha.gnu.org/gnu/wget/wget2-1.99.1.tar.gz"
+  sha256 "8d9b0b0897ff9623169db47c656d09007677f8d51de5f5cd3652258aab8b4dde"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext"
+  depends_on "gnutls" => :recommended
```
Comment (lembacon) [inline Formula/wget2.rb]: Please remove all `=> :recommended`, options have all been removed from `Homebrew/homebrew-core`.

File: Formula/wget2.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Wget2 < Formula
+  desc "Multithreaded metalink/file/website downloader"
+  homepage "https://gitlab.com/gnuwget/wget2/"
+  url "https://alpha.gnu.org/gnu/wget/wget2-1.99.1.tar.gz"
+  sha256 "8d9b0b0897ff9623169db47c656d09007677f8d51de5f5cd3652258aab8b4dde"
+
+  depends_on "autoconf" => :build
```
Comment (lembacon) [inline Formula/wget2.rb]: `autoconf` and `automake` do not seem necessary here as the tarball has already been autoconfed.

Comment (jakob-stoeck) [reply Formula/wget2.rb]: You’re right. The git repo needs them, the release tarball not anymore. Removed.

File: Formula/wget2.rb (hunk 1 above)
Comment (jakob-stoeck) [reply Formula/wget2.rb]: done

**Comment by fxcoudert** [comment]
> GNU Wget2 Release 1.99.1 (alpha)

alpha software is not acceptable in Homebrew: https://docs.brew.sh/Acceptable-Formulae

**Comment by jakob-stoeck** [comment]
Makes sense! Added it in a tap here for people who still want to use it since it works great: https://github.com/jakob-stoeck/homebrew-unacceptable

**Pull request closed without merging**